SetCurrentVersion.cmd

```diff
@@ -1,4 +1,4 @@
 set MAJOR=1
-set MINOR=4
-set PATCH=34
+set MINOR=5
+set PATCH=0
```

Comment:
This feels like more than a patch, but I'm willing to go back to 1.4.35 if you prefer.